src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form__big"
+      data-qa="big"
```

Comment:
HTML Formatting: The attributes of the `<form>` element should each start on a new line with 2-space indentation, and the closing bracket should be aligned with the opening tag. This will improve readability and maintain consistency with proper HTML formatting.